SCClassLibrary/Common/Streams/FilterPatterns.sc

```diff
@@ -445,11 +445,11 @@ Pfindur : FilterPattern {
 }
 
 Psync : FilterPattern {
-	var <>quant, <>maxdur, <>tolerance;
-	*new { arg pattern, quant, maxdur, tolerance = 0.001;
-		^super.new(pattern).quant_(quant).maxdur_(maxdur).tolerance_(tolerance)
+	var <>quant, <>maxdur, <>tolerance, <>mindur;
```

Comment:
Ok, I see the reason for `mindur`.

My only suggestion here is to document the new argument.

The ordering is unfortunate but alternatives are worse.